ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml

```diff
@@ -99,6 +90,14 @@ tests:
     from: src
 - as: sanity
   commands: |
+    echo "Debug statements"
+    pwd
+    id
+    echo "setting environment"
+    export GO111MODULE=on
+    export GOFLAGS=-mod=vendor
```

Comment:
We may have already talked about this, but ...

Do we need to explicitly set `-mod=vendor`? I feel like I remember that being the default in openshift/release go builder images, and only having to override it if we wanted something else.

Reply:
Removed both of these.